pkg/sql/catalog/lease/lease.go

```diff
@@ -73,6 +73,56 @@ var LeaseJitterFraction = settings.RegisterFloatSetting(
 	base.DefaultDescriptorLeaseJitterFraction,
 	settings.Fraction)
 
+//go:generate stringer -type=SessionBasedLeasingMode
```

Comment:
super nit: I think it's generally convention to add an extra newline here such that go:generate doesn't get picked up as the doc. string?